downstream/modules/eda/ref-eda-controller-install-builder.adoc

```diff
@@ -0,0 +1,11 @@
+[id="eda-controller-install-builder"]
+
+= Installing ansible-builder
+
+To build images, you must have Podman or Docker installed, along with the `ansible-builder` Python package. 
+
+The `--container-runtime` option must correspond to the Podman or Docker executable you intend to use.
+
+When building a decision environment image, it must support the architecture that {PlatformNameShort} is deployed with.
+
+For more information, see link:https://ansible.readthedocs.io/projects/builder/en/latest/#quickstart-for-ansible-builder[Quickstart for Ansible Builder] or link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/creating_and_using_execution_environments/index[Creating and consuming execution environments].
```

Comment:
Suggest using the link URL for the builder guide:

link:{LinkBuilder}.

Reply:
Thanks, @dcdacosta. Correction in progress.